run/system_package/graphviz.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2019 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Sample graphviz-web is a Cloud Run service which provides a CLI tool over HTTP.
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func main() {
+	// Verify the the dot utility is available at startup
+	// instead of waiting for a first request.
+	fileInfo, err := os.Stat("/usr/bin/dot")
```

Comment:
That is the location the graphviz package installs the dot utility in this OS. Because containerized applications tend to be more immutable, it's my preference to be explicit wherever possible. I've avoided making this configurable for sample readability.